src/core/src/Context.cpp

```diff
@@ -689,4 +696,17 @@ void Context::addGlobeAnchorToPrim(const pxr::SdfPath& path, double latitude, do
     globeAnchor.GetGeographicCoordinateAttr().Set(coordinates);
 }
 
+void Context::addCartographicPolygonPrim(const pxr::SdfPath& path) {
+    auto stage = Context::instance().getStage();
+    auto cartographicPolygon = pxr::CesiumCartographicPolygon::Define(stage, path);
+
+    // Until we support multiple georeference points, we should just use the default georeference object.
+    auto georeferenceOrigin = UsdUtil::getOrCreateCesiumGeoreference();
+
+    if (cartographicPolygon) {
+        pxr::CesiumGlobeAnchorAPI globeAnchorAPI(cartographicPolygon.GetPrim());
+        globeAnchorAPI.GetGeoreferenceBindingRel().AddTarget(georeferenceOrigin.GetPath());
+    }
+}
+
```

Comment:
Adam mentioned this might not be possible unless we implement globe anchors without Native, because defining globe anchors requires interfacing with Native at the C++ level.